src/cloudConsoleLauncher.ts

```diff
@@ -90,6 +90,24 @@ async function createTerminal(accessToken: string, armEndpoint: string, userSett
 	});
 }
 
+export async function getStorageAccountKey(
+	resourceGroup: string,
+	subscriptionId: string,
+	accessToken: string,
+	storageAccountName: string) {
+	return request({
```

Comment:
- Why not use Azure SDK?
- So we don't support other Azure instances such as mooncake etc.?

Reply:
That's the issue i wanted to call out to @zikalino  as well. Using REST API is straight forward but how do we address issues that @xscript pointed out?

Reply:
@yaweiw yes, i am no longer so sure whether it's a good idea (regarding querying resource group arm template as well).
i am not even sure whether using azure SDK from the extension is correct approach? we can address issues like this using Azure SDK. What if user doesn't have service principal. it also may get messy.

Reply:
i just wonder if it's possible to create a separate SSH session to Cloudshell (not in terminal)

Reply:
actually, i think i am ok with this. aren't we in control where cloudshell is created? it will never be created in mooncake, right?
so i think at this point i am ok with this solution and we shouldn't spend more time changing this.

Reply:
below reason to keep current rest behavior:
1. in current code, only info we got to auth with azure is token, it doesn't work with exsiting managment sdk which requires client_id, client_key except rest api.
2. cloud shell sdk team Chris mentioned he'll release an session property which contains storage info in an experiment api. The whole CloudShellLauncher.ts will need revisit to be replaced by that API once ready. https://github.com/Microsoft/vscode-azure-account/issues/34#issuecomment-372226035